ingestion_tools/poetry.lock

```diff
@@ -35,102 +35,102 @@ files = [
 
 [[package]]
 name = "aiohttp"
-version = "3.10.6"
```

Comment:
I believe we aren't updating the related toml file. So is this lock file update intended?

Reply:
Hmm I merged main into my branch and for some reason this shows up here. I'll remove from this PR